<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seismic_manager_Dialog</class>
 <widget class="QDialog" name="seismic_manager_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seismic Data Manager</string>
  </property>
  <widget class="QListWidget" name="data_list_Widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="seismic_info_textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Seismic data in survey directory</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>seismic_manager_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
